hrjobcontract/scss/civihr/_bootstrap-variables.sass

```diff
@@ -864,3 +871,4 @@ $text-muted:                  $gray
 // $dl-horizontal-offset:        $component-offset-horizontal
 //** Horizontal line color.
 // $hr-border:                   $gray-lighter
+$crm-form-control-shadow:     inset 0 0 3px 0 rgba(0,0,0,0.2)
```

Comment:
This PR should be only about colors, why are we amending box shadows?

Reply:
We have done this changes to make input boxes and panels as per shoreditch which were slightly different.